README.md

```diff
@@ -148,3 +148,7 @@ Example:
 ```ts
 new WebExtensionPlugin({ hmrConfig: false })
 ```
+
+### <a id="options-weakRuntimeCheck"></a>`options`.`weakRuntimeCheck`
```

Comment:
So I suppose the solution suggested by this PR is to enable this option as a workaround to the weird CSS plugin behavior?

Reply:
yes. we need `browser.runtime` and if it is missing, it will throw. And the mini-css-extract-plugin will execute the generated bundle in NodeJS therefore it failed to build.

I give a weaker check if `self.addEventListener` does not exist (they exist in Webpage and Workers).

Reply:
how do you think this solution?

Reply:
From my understanding, isChrome/noChrome is normally executed in the browser _immediately_ even though `import()` is never used in the bundle. Is that correct?

If so, can these checks made lazy as well? i.e. run them only when they're required. Would this alone fix the CSS plugin build error?

If so, I think it's worth making that change instead of adding an extra option. Caching the result of these variables is comparatively trivial, if that's the issue.

Reply:
> From my understanding, isChrome/noChrome is normally executed in the browser immediately even though import() is never used in the bundle. Is that correct?

Actually not, the chunk loading (even for initial chunks) also requires `chrome`. And autoPublicPath also needs that.